src/subdomains/supporting/payment/dto/transaction-helper/tx-fee-details.dto.ts

```diff
@@ -1,6 +1,8 @@
+import { PriceStep } from 'src/subdomains/supporting/pricing/domain/entities/price';
 import { FeeDto, InternalFeeDto } from '../fee.dto';
 
 export interface TxFeeDetails {
   minVolume: number;
   fee: InternalFeeDto & FeeDto;
+  priceSteps: PriceStep[];
 }
```

Comment:
This could then be removed again

Reply:
- For BuyFiat, always inputReferenceAsset is always the same as inputAsset
- For BuyCrypto, they might be different but the price source fir conversion from input- to inputReferenceAsset is our bank